docs/fx-core/spfx-help-v5.md

```diff
@@ -1,2 +1,21 @@
 # SPFx troubleshoot
 
+1. [Failure to scaffold](#scaffold)
```

Comment:
Scaffolding errors / Scaffolding failures / Failed to scaffold. Either one could work.

Reply:
updated.